packages/preview-middleware/src/base/flp.ts

```diff
@@ -196,16 +196,18 @@ export class FlpSandbox {
             },
             locateReuseLibsScript: this.config.libs ?? (await this.hasLocateReuseLibsScript())
         };
+        const id = manifest['sap.app'].id;
         this.addApp(manifest, {
             componentId,
-            target: resources[componentId ?? manifest['sap.app'].id] ?? this.templateConfig.basePath,
+            target: resources[componentId ?? id] ?? this.templateConfig.basePath,
             local: '.',
             intent: this.config.intent
         });
         this.addStandardRoutes();
         if (this.rta) {
             this.rta.options ??= {};
-            this.rta.options.baseId = componentId ?? manifest['sap.app'].id;
+            this.rta.options.baseId = componentId ?? id;
+            this.rta.options.appName = manifest['sap.app'].title ?? id;
```

Comment:
This takes the title from the deployed app and from the adp prokject. As a result in the browser we see `Editing {{appTitle}}` instead of `Editing My Application`

Reply:
Not sure if this is correct: [ef164f5](https://github.com/SAP/open-ux-tools/pull/1553/commits/ef164f5d741d8744ebae0aab55c6ac41e8bdf4a8) I. But now if we have the title from the manifest as {{appTitle}} we use 'My Application' instead.

Reply:
As discussed we will just show the id as appTitle